common/ui/menubar.cpp

```diff
@@ -751,8 +751,12 @@ void menubar_init( const char * file )
 			j.Hotkey = -1;
 	auto infile = PHYSFSX_openReadBuffered(file);
 
-	if (!infile) return;
-		
+	if (!infile)
+	{
+		Warning("Couldn't find %s\n", file);
```

Comment:
This commit adds several `Warning` calls of this form.  Some of them use `Couldn't`; others use `Could not`.  It would be nice to be consistent.